server/src/main/java/io/druid/server/coordinator/DruidCluster.java

```diff
@@ -70,6 +72,15 @@ public void add(ServerHolder serverHolder)
     return cluster.get(tier);
   }
 
+  public Set<ServerHolder> getAllServers()
+  {
+    final Set<ServerHolder> servers = new HashSet<>();
```

Comment:
@gianm I found ```ServerHolder.equals()``` calls ```ImmutableDruidServer.equals()``` and ```LoadQueuePeon.equals()``` which are not implemented. Maybe this is not a critical problem for now because it is not called so far.. Anyway, I'll fix it. 
Btw, to check the equality of two ```ServerHolder```s, I think it would be enough to compare ImmutableDruidServer.getHost(), ImmutableDruidServer.getType(), and ImmutableDruidServer.getTier(). What do you think?